src/RollOfHonour.Core/Models/Search/SearchQuery.cs

```diff
@@ -0,0 +1,12 @@
+using RollOfHonour.Core.Enums;
+
+namespace RollOfHonour.Core.Models.Search;
+
+public class SearchQuery
```

Comment:
This class seems a little  unsure of itself.  It has a QueryType, which could be memorial.  At that point PersonType do you feel Person Type is appropriate?  Could more specific types to this job more purposefully?

Reply:
This has been done.